lib/chewy.rb

```diff
@@ -1,3 +1,4 @@
+require 'active_support'
```

Comment:
I confirm, it's pretty safe https://guides.rubyonrails.org/active_support_core_extensions.html#stand-alone-active-support